Tensile/SolutionLibrary.py

```diff
@@ -134,14 +96,52 @@ def merge(self, other):
 
         self.table.sort(key=lambda r: r['key'])
 
-    def remapSolutionIndices(self,indexMap):
+    def remapSolutionIndices(self, indexMap):
         pass
 
     def __init__(self, properties, table, distance):
         self.properties = properties
         self.table = table
         self.distance = distance
 
+
+class DecisionTreeLibrary:
+    Tag = 'DecisionTree'
+    StateKeys = [('type', 'tag'), 'properties', 'trees']
+
+    @classmethod
+    def FromOriginalState(cls, d, solutions):
+        properties = d["properties"]
+        origTrees = d["trees"]
+
+        trees = []
+
+        for tree in origTrees:
+            index = tree["solution"]
+            value = SingleSolutionLibrary(solutions[index])
+
+            entry = {'tree': tree["tree"], 'value': value}
```

Comment:
I had meant to fix this before posting the PR but forgot; thanks for the reminder. I decided on double quotes for the file since that seems to be the more common choice in other files.